tests/unit/conf/global.json

```diff
@@ -29,13 +29,33 @@
     "dynamodb_table": "test_table_name",
     "enabled": false,
     "excluded_iocs": {
-      "ip": [
-        "127.0.0.0/8",
-        "10.0.0.0/8",
-        "172.16.0.0/12",
-        "192.168.0.0/16",
-        "52.52.52.52/32"
-      ]
+      "domain": {
+        "not-evil.com": {
```

Comment:
Can we just use
```
"domain": {
  "not-evil.com": "This domain is not evil"
}
```

We can discuss more if we have plan to add more keys other than `comment`.

Reply:
this is possible, and more readable/writable, i guess the discussion is a good thing to have regarding whether or not we will associate more metadata than just the comment

Reply:
I like the flexibility the current version affords us; ill bet we'll want to add more metadata in the future

Reply:
Make sense then. Thanks for the clarification. 😸